data/json/recipes/food/distill.json

```diff
@@ -0,0 +1,83 @@
+[
+    {
+        "type" : "recipe",
+        "result": "brandy",
+        "category": "CC_FOOD",
+        "subcategory": "CSC_FOOD_DRINKS",
+        "skill_used": "cooking",
+        "difficulty": 3,
+        "time": 30000,
+        "batch_time_factors": [50, 4],
+        "autolearn": [ [ "cooking", 4 ] ],
+        "book_learn": [ [ "brewing_cookbook", 2 ] ],
+        "qualities": [
+            { "id": "BOIL", "level": 2 },
+            { "id": "DISTILL", "level": 2 }
+        ],
+        "tools": [ [ [ "hotplate", 10 ], [ "char_smoker", 10 ], [ "toolset", 10 ], [ "fire", -1 ] ] ],
```

Comment:
Distillation generally requires heating a given volume to a certain temperature. As a result the energetic requirements are likely to be comparable to those used in the ethanol recipes. 10 charges is too cheap for what is an energy-intense process.